### PR TITLE
Implemented Profile Screen With Firebase Integration

### LINKED ISSUE #20: Firebase Profile Screen

### PR DESCRIPTION
**Pull request by AbeerAdeel**

### CODE REVIEW

File: lib/models/UserModel.dart [hunk 1]
```diff
@@ -18,22 +14,27 @@ class UserModel {
   Gender gender = Gender.unspecified;
   String? age;
   String? weight;
-  String? heightFeet;
-  String? heightInches;
+  String? height;
+  String imageURL =
+      "https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png?20150327203541";
+  String fitnessStyle = "default";
```
Comment (AbeerAdeel) [inline lib/models/UserModel.dart:20]: This is just the the default person icon photo

File: lib/Screens/profile.dart [hunk 2]
```diff
@@ -361,7 +517,25 @@ class _EditProfileState extends State<EditProfile> {
...
+                          widget.profile["desiredWeight"],
+                          widget.profile["fitnessStyle"],
+                          widget.profile['height'],
+                          widget.profile['gender'],
+                          widget.profile['imageURL'],
+                        ),
+                      );
+                      if (isWeightChanged) {
+                        await weightController.addWeight(WeightModel(
+                          widget.profile["weight"],
```
Comment (AbeerAdeel) [inline lib/Screens/profile.dart:535]: Add weight to metrics

**Review by jjchester — COMMENTED** [review]
Code looks good but when I'm logged into mine I can't see the follower/following list even tho the number is 1 for each.

**Comment by jjchester** [comment]
LGTM

**Pull request merged**